grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -106,6 +109,43 @@ class ApiController extends BaseController {
         }
     }
 
+    @GrantAccess
+    @Path("/api/opus")
+    @Operation(
+            summary = "Get all public collections",
+            operationId = "/api/opus",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    array = @ArraySchema(
+                                            schema = @Schema(
+                                                    implementation = OpusResponse.class
```

Comment:
Replace with CollectionList since OpusResponse has a lost more properties than what is returned by this API.